src/chain/payout.rs

```diff
@@ -76,8 +76,33 @@ pub async fn payout(
                 }
             },
             a if (loss_tolerance..=manual_intervention_amount).contains(&a) => {
-                tracing::warn!("Overpayments not handled yet");
-                return Ok(()); //TODO
+                tracing::warn!("Overpayment, proceeding with available balance");
+                // We will transfer all the available balance
```

Comment:
```suggestion
                // We will transfer all the available balance
                // TODO smarter handling and returns probably
```

Reply:
resolved in [16fe114](https://github.com/Alzymologist/Kalatori-backend/pull/106/commits/16fe114e333990884ec4beabb69f1e5518c74cde)